After Aer .14 is released, remove approximation=true from relevant pages (#1020)
Qiskit/documentation: docs/api/migration-guides/qiskit-algorithms-module.mdx

* Running an algorithm on an Aer simulator using a custom instruction.  For example, using the [`qiskit.opflow`](../qiskit/0.46/opflow) legacy
[`qiskit.opflow.expectations.AerPauliExpectation`](../qiskit/0.46/qiskit.opflow.expectations.AerPauliExpectation).

> - Aer Primitives with `shots=None`.  See [TrotterQRTE](#trotterqrte) for examples.

Reviewer: ```suggestion
- Aer Primitives with default options.  See [TrotterQRTE](#trotterqrte) for examples.
```

```python
from qiskit_aer.primitives import Sampler, Estimator

> sampler = Sampler(run_options={"shots": None})
> estimator = Estimator(run_options={"shots": None})

Reviewer: ```suggestion
sampler = Sampler()
estimator = Estimator()
```

I think this is `None` by default:

```python
>>> from qiskit_aer import Estimator
>>> print(Estimator().options.get("shots"))
None
```

Author: Thanks!

from qiskit_aer.primitives import Sampler, Estimator

> sampler = Sampler(run_options={"approximation": True, "shots": None})

Author: @kevinsung the change is only for Estimator, not Sampler, correct? Should I leave this line as-is?

Reviewer 2: As far as I know, "approximation" is not a valid run option, so I think it can be removed immediately from all examples of Sampler and Estimator. See https://github.com/Qiskit/qiskit-aer/issues/2083 though.

https://github.com/Qiskit/documentation/issues/319 Refers to passing `approximation=True` directly to the initializer of Estimator, as in those pages in the verify section.

evolution_problem = TimeEvolutionProblem(operator, 1, initial_state)

# Aer simulator using custom instruction

> estimator = AerEstimator(run_options={"shots": None})

Reviewer: ```suggestion
estimator = AerEstimator()
```

Reviewer: Actually the "custom instruction" comment needs changing too
